apps/web/src/pages/changes/PromoteChangesPage.tsx

```diff
@@ -43,6 +43,7 @@ export function PromoteChangesPage() {
       value: PENDING,
       content: (
         <ChangesTable
+          key={page}
```

Comment:
The page number indicator at the bottom of the page sometimes will be out of sync with the current page number,
Adding a `key` fixed that issue